### DISCUSSION TITLE
I'm not getting useful results from scalene, help! (Thread.run)

### DISCUSSION THREAD

**Discussion by ypearson** [plasma-umass/scalene#826] [Q&A] [answered]
![image](https://github.com/plasma-umass/scalene/assets/13055532/4c814d61-4a5d-4ced-b91e-6f070a10e06c)

This is all I get, which is not useful for me. This is not the code that I'd like to exam and profile.

`scalene --reduced-profile --profile-all test_script.py`

**Comment by emeryberger** [comment, marked as answer]
Try adding `--profile-exclude threading` (or even `--profile-exclude python3.10` to eliminate any libraries).

**Reply by ypearson** [reply]
Thx

**Comment by ypearson** [comment]
This is exactly what I'm looking for! Thank you. I can't test it today, but will do on Monday.
